### PR TITLE
qa/dashboard: disable HEALTH check in test_pool.test_pool_update_metadata

### PR DESCRIPTION
**Pull request by sebastian-philipp**
We're getting suspicious HEALTH_WARN errors here.

Fixes: https://tracker.ceph.com/issues/45717

Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/tasks/mgr/dashboard/test_pool.py [hunk 1]
```diff
@@ -315,22 +316,26 @@ def test_pool_update_metadata(self):
             props = {'application_metadata': ['rbd', 'sth']}
             self._task_put('/api/pool/{}'.format(pool_name), props)
             time.sleep(5)
-            self._validate_pool_properties(props, self._get_pool(pool_name))
+            # disabled HEALTH check, due to suspicious HEALTH_WARN errors
+            self._validate_pool_properties(props, self._get_pool(pool_name), validate_health=False)
```
[the remark this answers is not in the document]
Comment (sebastian-philipp) [reply qa/tasks/mgr/dashboard/test_pool.py:320]: you want to make a PR for that?

[the remark this answers is not in the document]
Comment (epuertat) [reply qa/tasks/mgr/dashboard/test_pool.py:320]: If you don't mind...

[the remark this answers is not in the document]
Comment (sebastian-philipp) [reply qa/tasks/mgr/dashboard/test_pool.py:320]: no! I mean, you already know what you want to do to improve this specific behavior, I was just guessing here. feel free to simply close this PR!

[the remark this answers is not in the document]
Comment (epuertat) [reply qa/tasks/mgr/dashboard/test_pool.py:320]: Ok, will do.

**Pull request closed without merging**